### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 0mar-K**
Added 5 examples for Annotations
Created a new section for Form Creator and added 3 examples

Looping in @ChrisixFlash for review

### CODE REVIEW

**Review by jdrhyne — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -43,6 +48,12 @@ This repository contains examples built with PSPDFKit. These examples are not of
...
...Cx7c3VidHJlZTohMCxjaGlsZExpc3Q6ITB9KX1pbnN0YW5jZS5hZGRFdmVudExpc3RlbmVyKCdhbm5vdGF0aW9ucy5wcmVzcycsKGV2ZW50PT57XG4vL3RoaXMgd2lsbCBtYWtlIHN1cmUgd2Ugc2V0IHRoZSBvYnNlcnZlciBldmVyeSB0aW1lIGEgZGF0ZSB3aWRnZXQgaXMgY2xpY2tlZCBpbiBmb3JtIGNyZWF0b3IgbW9kZS5cbmNvbnN0e2ludGVyYWN0aW9uTW9kZTppbnRlcmFjdGlvbk1vZGV9PWluc3RhbmNlLnZpZXdTdGF0ZTtjb25zb2xlLmxvZyhpbnRlcmFjdGlvbk1vZGUpLCdGT1JNX0NSRUFUT1InPT09aW50ZXJhY3Rpb25Nb2RlJiZzZXRPYnNlcnZlcigpfSkpLGluc3RhbmNlLmFkZEV2ZW50TGlzdGVuZXIoJ3ZpZXdTdGF0ZS5jaGFuZ2UnLCgoY3VycmVudCxwcmV2KT0%252Be1xuLy9oZXJlIHdlIG1ha2Ugc3VyZSB3ZSBzZXQgdGhlIGxvY2FsZSBpZiB0aGVyZSdzIG9uZSBzZXQgYWxyZWFkeVxuY29uc3R7aW50ZXJhY3Rpb25Nb2RlOmludGVyYWN0aW9uTW9kZX09Y3VycmVudCxsb2NhbGU9bG9jYWxTdG9yYWdlLmdldEl0ZW0oJ2RhdGVfcGlja2VyX2Zvcm1hdCcpOydEQVRFX1dJREdFVCc9PT1pbnRlcmFjdGlvbk1vZGUmJihzZXRPYnNlcnZlcigpLGxvY2FsZSYmaW5zdGFuY2Uuc2V0T25XaWRnZXRBbm5vdGF0aW9uQ3JlYXRpb25TdGFydCgoYW5ub3RhdGlvbj0%252BKHthbm5vdGF0aW9uOmFubm90YXRpb24uc2V0KCdhZGRpdGlvbmFsQWN0aW9ucycse29uRm9ybWF0Om5ldyBQU1BERktpdC5BY3Rpb25zLkphdmFTY3JpcHRBY3Rpb24oe3NjcmlwdDpgQUZEYXRlX0Zvcm1hdEV4KFwiJHtKU09OLnBhcnNlKGxvY2FsZSl9XCIpYH0pfSl9KSkpKX0pKX0pKTsiLCJtb2RlIjoic3RhbmRhbG9uZSIsImNzcyI6Ii8qIEFkZCB5b3VyIENTUyBoZXJlICovXG4ifQ%253D%253D)
```
Comment (ChrisixFlash) [inline README.md]: Let's skip this, discussed it with Eli and it makes little sense to add.

File: README.md [hunk 2]
```diff
@@ -21,6 +21,11 @@ This repository contains examples built with PSPDFKit. These examples are not of
...
...GhlbigoYXN5bmMgaW5zdGFuY2U9Pntcbi8vZ2VuZXJhdGluZyB0aGUgYmF0ZXMgdXNpbmcgdGhlIHBhZ2UgaW5mbyBkaW1lbnNpb25zXG5sZXQgYmF0ZXM9KGF3YWl0IFByb21pc2UuYWxsKEFycmF5LmZyb20oe2xlbmd0aDppbnN0YW5jZS50b3RhbFBhZ2VDb3VudH0pLm1hcCgoKF8scGFnZUluZGV4KT0%252BaW5zdGFuY2UucGFnZUluZm9Gb3JJbmRleChwYWdlSW5kZXgpKSkpKS5yZWR1Y2UoKChhY2MscGFnZWluZm8pPT57Y29uc3R7aW5kZXg6aW5kZXgscmF3UGRmQm94ZXM6cmF3UGRmQm94ZXN9PXBhZ2VpbmZvLHtjcm9wQm94OmNyb3BCb3h9PXJhd1BkZkJveGVzLFtsZWZ0LHRvcCx3aWR0aCxoZWlnaHRdPWNyb3BCb3gsdmFsdWU9UFNQREZLaXQuZ2VuZXJhdGVJbnN0YW50SWQoKTtyZXR1cm4gYWNjLmNvbmNhdChuZXcgUFNQREZLaXQuQW5ub3RhdGlvbnMuVGV4dEFubm90YXRpb24oe3BhZ2VJbmRleDppbmRleCx0ZXh0Ontmb3JtYXQ6J3BsYWluJyx2YWx1ZTp2YWx1ZS50b1N0cmluZygpfSxmb250OidIZWx2ZXRpY2EnLGZvbnRTaXplOjExLGlzQm9sZDohMCxob3Jpem9udGFsQWxpZ246J2NlbnRlcicsYm91bmRpbmdCb3g6bmV3IFBTUERGS2l0Lkdlb21ldHJ5LlJlY3Qoe2xlZnQ6d2lkdGgtMTEqdmFsdWUudG9TdHJpbmcoKS5sZW5ndGgsdG9wOjIwLHdpZHRoOjExKnZhbHVlLnRvU3RyaW5nKCkubGVuZ3RoLGhlaWdodDo0MH0pLGZvbnRDb2xvcjpQU1BERktpdC5Db2xvci5SRUR9KSl9KSxbXSk7YXdhaXQgaW5zdGFuY2UuY3JlYXRlKGJhdGVzKSxhd2FpdCBpbnN0YW5jZS5hcHBseU9wZXJhdGlvbnMoW3t0eXBlOidmbGF0dGVuQW5ub3RhdGlvbnMnfV0pfSkpOyIsIm1vZGUiOiJzdGFuZGFsb25lIiwiY3NzIjoiLyogQWRkIHlvdXIgQ1NTIGhlcmUgKi9cblx0In0%253D)
```
Comment (ChrisixFlash) [inline README.md]: This we can also skip as it's already being worked on internally: https://pspdfkit.com/guides/web/editor/add-bates-number/

**Review by ChrisixFlash — CHANGES_REQUESTED** [review]
Let's remove these 2 and merge the rest!

**Review by ChrisixFlash — APPROVED** [review]
Removed the examples I mentioned, good to merge now :)

**Pull request merged**